docs/pages/2020_Intro_Week/sections/workflows_adv.rst

```diff
@@ -252,6 +284,19 @@ This utility now allows us to simplify the ``setup`` outline step that we have s
 
 This way we don't have to manually fish out all the individual inputs from the ``self.inputs`` but have to just call this single method, saving a lot of time and lines of code.
 
+When submitting or running the work chain using namespaced inputs (``add`` in the example above), it is important to use the namespace:
+
+.. code-block:: python
+
+    submit(ArithmeticAddBaseWorkChain, **{'add': {'x': Int(3), 'y': Int(4), 'code': load_code('add@localhost')}})
+
+.. note::
+
+    When updating an existing work chain file or adding a new one, it is **necessary** to restart the daemon **every time** after all changes have taken place:
+
+    .. code-block:: bash
+
+        $ verdi daemon restart --reset
```

Comment:
The location of this feels a bit weird. Why not include the statement that you also have to restart when having changed files in the one where you talk about the python path